confidential-data-hub/hub/src/bin/confidential-data-hub/main.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{path::Path, sync::Arc};
+
+use anyhow::{Context, Result};
+use api_ttrpc::create_sealed_secret_service;
+use clap::Parser;
+use log::info;
+use server::Server;
+use tokio::{
+    fs,
+    signal::unix::{signal, SignalKind},
+};
+use ttrpc::r#async::Server as TtrpcServer;
+
+use crate::api_ttrpc::create_get_resource_service;
+
+mod api;
+mod api_ttrpc;
+mod server;
+
+const DEFAULT_UNIX_SOCKET_DIR: &str = "unix:///run/confidential-containers";
```

Comment:
a spelling mistake, dir should be /run/confidential-containers.

Reply:
Thanks, fixed this